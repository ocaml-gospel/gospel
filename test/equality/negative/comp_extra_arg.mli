type 'a t

val f : (int -> int -> int) -> 'a t -> 'a t -> int
(*@ comparison *)
