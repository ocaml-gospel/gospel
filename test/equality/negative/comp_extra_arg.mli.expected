
*******************************
********** Parsed file ********
*******************************
type 'a t
val f : (int -> int -> int) -> 'a t -> 'a t -> int[@@gospel {| comparison |}]

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Gospelstdlib *)

type 'a t
  

val f : (int -> int -> int) -> 'a t -> 'a t -> int
(*@ comparison
     *)
File "comp_extra_arg.mli", lines 3-4, characters 0-27:
Error: Type checking error: type is incompatible with equality clause
